### ISSUE TITLE
[afr] typo?

### ISSUE THREAD

**Issue by eric-muller** [unicode-org/udhr#100]
The document contains ho&#x2019;r and materi&#x2019;le. Should these be
ho&#x00EB;r and materi&#x00EB;le?

**Comment by moyogo** [comment]
Yes, those would be the correct spellings. The translation on [Afrikaans.us](https://www.afrikaans.us/afrikaans/pronunciation/holistic-focus/lees/verklaring-van-menseregte/) has "hoër opvoeding" and "materiële belange". 

There is also "materiële belange" in *Opsomming van aanbevelings vir Suid-Afrika deur die Verenigde Nasies se Komitee oor Ekonomiese, Maatskaplike en Kulturele Regte*, 2019 ([PDF](https://admin.dullahomarinstitute.org.za/socio-economic-rights/research-and-publications/publications/afr-otb8429-uwc-dullah-omarcescr-19-july-2019-1.pdf#page=4)) which rephrases some parts of the UDHR.
